academic-services/knowledge-exploration-service/reference-get-interpret.md

```diff
@@ -0,0 +1,459 @@
+---
+title: GET Interpret
+description: Generates semantic interpretations of a natural language query
+ms.topic: reference
+ms.date: 2020-02-07
+---
+
+# Interpret REST API
+
+The **Interpret** method generates semantic interpretations of a natural language query based on the combination of a language grammar and indexed attributes.
+
+It can be used as either a way of interpreting a specific natural language query, or to generate natural language query suggestions using the original query as a stem.
+
+For full details on the language grammar see the [natural language queries](concepts-queries.md) page.
+
+``` HTTP
+GET http://{serviceName}.{serviceRegion}.cloudapp.azure.com/interpret?query={query}
+```  
+
+With optional parameters:
+
+``` HTTP
+GET http://{serviceName}.{serviceRegion}.cloudapp.azure.com/interpret?query={query}&count={count}&offset={offset}&timeout={timeout}&complete={complete}&entityCount={entityCount}&attributes={attributes}
+```  
+
+## URI Parameters
+
+Name | Required | Type | Description
+--- | --- | --- | ---
+`query` | Required | string | The natural language query to generate interpretations for. See the [natural language queries](concepts-queries.md) page for more details.
+`count` | Optional | integer | The number of interpretations to generate in the result set. <br/><br/>Defaults to 5.
```

Comment:
>The number of interpretations [](start = 31, length = 29)

Is it the "max" number of interpretations?

Reply:
Not sure what you mean by "max"

---
In reply to: [373267164](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/97#discussion_r373267164) [](ancestors = 373267164)